Sources/ProjectSpec/Target.swift

```diff
@@ -57,7 +58,8 @@ public struct Target: ProjectTarget {
     public var productName: String
     public var onlyCopyFilesOnInstall: Bool
     public var putResourcesBeforeSourcesBuildPhase: Bool
-
+    public var isResolved: Bool
```

Comment:
Could you add a code comment here for future users. The name isn't very specific